src/pages/dashboard/index.tsx

```diff
@@ -33,12 +34,12 @@ function ProjectRecommendations({ projects }: { projects: any[] }) {
       <CardContent>
         <div className="space-y-4">
           {projects.map((project, index) => (
-            <a key={index} href={project.htmlUrl} target="_blank" rel="noopener noreferrer">
-              <div key={index} className="pb-4 mb-4 border-b">
+            <Link key={index} to={`/projects/${project.name}/tasks`}>
```

Comment:
Please refer project list implementation. Keep both link to github and to task list page.

Reply:
ok